src/Framework/FieldsAPI/Checkbox.php

```diff
@@ -12,6 +12,7 @@ class Checkbox extends Field {
 	use Concerns\HasLabel;
 	use Concerns\HasPlaceholder;
 	use Concerns\ShowInReceipt;
+	use Concerns\HasOptions;
```

Comment:
I’ve been trying to keep these sorted alphabetically.

```suggestion
	use Concerns\HasOptions;
	use Concerns\HasPlaceholder;
	use Concerns\ShowInReceipt;
```